python/sglang/srt/server.py

```diff
@@ -250,6 +251,10 @@ def launch_server(
     )
     logger.info(f"{server_args=}")
 
+    # Use model from www.modelscope.cn, first download the model.
+    server_args.model_path = prepare_model(server_args.model_path)
+    server_args.tokenizer_path = prepare_model(server_args.tokenizer_path)
```

Comment:
For tokenizer, we should not download weights.

Reply:
ignore weight files.